### PR TITLE
[FrameworkBundle] Incorrect line break in exception message (500 debug page)

### PR DESCRIPTION
**Pull request by pedroresende**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | [19041](https://github.com/symfony/symfony/issues/19041) |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css [hunk 1]
```diff
@@ -39,7 +39,7 @@ build: 56
     font-family: Georgia, "Times New Roman", Times, serif;
     font-size: 20px;
     color: #313131;
-    word-break: break-all;
+    word-break: keep-all;
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css]: wouldn't it be better to use `word-wrap: break-word` here?

Comment (pedroresende) [inline src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css]: @nicolas-bastien You're right, already updated to break-word and rebased the PR

File: src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css [hunk 2]
```diff
@@ -39,7 +39,7 @@ build: 56
     font-family: Georgia, "Times New Roman", Times, serif;
     font-size: 20px;
     color: #313131;
-    word-break: break-all;
+    word-break: break-word;
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css]: Should be `word-wrap: break-word`: `break-word` is an unofficial value for `word-break`.

Comment (pedroresende) [inline src/Symfony/Bundle/FrameworkBundle/Resources/public/css/body.css]: @nicolas-bastien you're right, http://www.w3.org/TR/css-text-3/#word-wrap

**Comment by fabpot** [comment]
Thank you @pedroresende.

**Comment by pedroresende** [comment]
@fabpot Sure thing, always glad to help a wonderful project. Btw any special reason for not being added to the list of contributors ?

**Comment by jakzal** [comment]
@pedroresende that list is refreshed periodically. You'll eventually get there :)

**Pull request closed without merging**